autoscale-docker-swarm-container/pom.xml

```diff
@@ -0,0 +1,253 @@
+                                        <args>-v</args>
+                                        <args>--</args>
+                                        <args>/maven/scaler.sh</args>
+                                    </exec>
+                                </entryPoint>
+                                <assembly>
+                                    <mode>tar</mode>
+                                    <inline>
+                                        <fileSets>
+                                            <fileSet>
+                                                <includes>
+                                                    <include>*.sh</include>
+                                                </includes>
+                                                <fileMode>0755</fileMode>
+                                            </fileSet>
+                                            <fileSet>
+                                                <includes>
+                                                    <include>*.yaml</include>
+                                                </includes>
+                                            </fileSet>
+                                            <fileSet>
+                                                <directory>src/main/config</directory>
+                                                <outputDirectory>config</outputDirectory>
+                                            </fileSet>
+                                        </fileSets>
+                                        <dependencySets>
+                                            <dependencySet>
+                                                <useProjectArtifact>false</useProjectArtifact>
+                                                <scope>runtime</scope>
+                                                <excludes>
+                                                    <exclude>com.github.krallin:tini</exclude>
+                                                </excludes>
+                                            </dependencySet>
+                                            <dependencySet>
+                                                <useProjectArtifact>false</useProjectArtifact>
+                                                <fileMode>0755</fileMode>
+                                                <includes>
+                                                    <include>com.github.krallin:tini</include>
+                                                </includes>
+                                            </dependencySet>
+                                        </dependencySets>
+                                    </inline>
+                                </assembly>
+                            </build>
+                            <run>
+                                <extraHosts>
+                                    <host>vagrant-mesos:192.168.56.10</host>
```

Comment:
set your DOCKER_HOST to http://vagrant-mesos:2375
Now try to run the container and it will fail, as the docker host env inside the container is trying to communicate with the machine via vagrant-mesos which isn't a known hosts ( as it isn't on dns ).  This adds the entry to allow a users machine toeither have http://vagrant-mesos or http://192.168.56.10